app/views/users/show.html.erb

```diff
@@ -5,6 +5,7 @@
 <p class='bio'><%= @user.bio %></p>
 <br>
 <% @post.each_with_index do |posts, idx|%>
+    <% break if idx == 3 %>
```

Comment:
- [ ] According to the previous reviewer, instead of creating new logic, please consider using the method you already defined in your user model: 🤓 

https://github.com/francksefu/WebApp/blob/a63e0e7a861dd92dac0a0fb3f3d3832d675f3d65/app/models/user.rb#L6-L9